docs/hdfs.md

```diff
@@ -5,16 +5,19 @@ menu_order: 20
 enterprise: 'no'
 ---
 
-To configure Spark for a specific HDFS cluster, configure `hdfs.config-url` to be a URL that serves your `hdfs-site.xml` and `core-site.xml`. For example:
+You can configure Spark for a specific HDFS cluster.
 
-    {
-      "hdfs": {
-        "config-url": "http://mydomain.com/hdfs-config"
-      }
-    }
+To configure `hdfs.config-url` to be a URL that serves your `hdfs-site.xml` and `core-site.xml`, use this example where `http://mydomain.com/hdfs-config/hdfs-site.xml` and `http://mydomain.com/hdfs-config/core-site.xml` are valid URLs:
 
+```json
```

Comment:
I think @sascala told me that these language indentifiers are Github specific, and don't work when we translate these to the service docs webpage.  Is that still the case?